pages/common/d8.md

```diff
@@ -0,0 +1,16 @@
+# d8
+
+> Developer shell for the V8 JavaScript engine.
+> More information: <https://v8.dev/docs/d8>.
+
+- Start a REPL (interactive shell):
+
+`d8`
+
+- Run a JavaScript file:
+
+`d8 {{path/to/file.js}}`
+
+- Evaluate a JavaScript expression:
+
+`d8 -e "{{code}}`
```

Comment:
Does this have --evaluate ?

Reply:
No. The documentation mentions no long form for `-e` and I just tried `--eval` and `--evaluate`, which both don't work.